xbmc/video/VideoDatabase.cpp

```diff
@@ -289,6 +289,8 @@ void CVideoDatabase::CreateAnalytics()
 
   m_pDS->exec("CREATE INDEX ix_videoversion ON videoversion (idMedia, mediaType(20))");
 
+  m_pDS->exec(PrepareSQL("CREATE INDEX ix_movie_title ON movie (c%02d)", VIDEODB_ID_TITLE));
+
```

Comment:
Just asking, will the index created without a database version bump?

Reply:
no you're right it won't...

Reply:
blessing in disguise actually, the index is fine in sqlite but based on previous PR mariadb probably won't like it without field length.